eventstore/migrations/0010_create_custom_types.up.sql

```diff
@@ -1,3 +1,8 @@
+-- "migration" written before we had proper migration handling, hence the
```

Comment:
Maybe we should add a comment in these files to say something like:

'NEVER update these files once they have been merged to main. If you need to make changes create a new change file'

Sadly I've seen far too many developers decide to try and change these files after the fact and create a mess.

Reply:
:scream: really?

Reply:
I've seen it happen about 5-10 times throughout my carrier sadly.

Reply:
Your carrier?

Have put a banner. Though it actually reminds me that something I _do_ sometimes do is add comments to old migrations when e.g. function definitions have been superseded in later migrations.